kind.sh

```diff
@@ -188,8 +189,14 @@ install_kind() {
 
     mkdir -p "${kind_dir}"
 
-    curl -sSLo "${kind_dir}/kind" "https://github.com/kubernetes-sigs/kind/releases/download/${version}/kind-linux-${arch}"
-    chmod +x "${kind_dir}/kind"
+    pushd "${kind_dir}"
+    wget --quiet "https://github.com/kubernetes-sigs/kind/releases/download/${version}/kind-linux-${arch}"
```

Comment:
why switch from curl to wget? this will break my workflow because no wget.

This is somehow a breaking change 🤔

Reply:
i don't remember exaclty, but would you like to move back to curl and open a pr?

Reply:
no, I added wget to my runner now 🤗

you should probably update release notes and add a note about the new wget requirements 🙃

Reply:
did this pr https://github.com/helm/kind-action/pull/110